src/doc/components/Esdoc/SummaryDoc.js

```diff
@@ -77,17 +82,24 @@ const SummaryDoc = ({
                     )}
                     <span data-ice="async">{doc.async ? 'async' : ''}</span>
                     <span data-ice="generator">{doc.generator ? '*' : ''}</span>
-                    <span className="code" data-ice="name">
-                      <DocLinkHTML
-                        longname={doc.longname}
-                        text={null}
-                        inner={innerLink}
-                        kind={doc.kind}
-                      />
-                    </span>
-                    <span className="code" data-ice="signature">
-                      <SignatureHTML doc={doc} />
-                    </span>
+                    {/* Custom return if 'constructor' */}
+                    {doc.kind === 'constructor' ? (
+                      `new ${doc.longname.match(/~([^)]+)#constructor/)[1]}()`
```

Comment:
Missing the link

Reply:
You mean, for example, this 'new TrackerLayer()' should be a link?

Reply:
'TrackerLayer' should be a link to the constructor